database/seeders/PermissionSeeder.php

```diff
@@ -14,6 +15,19 @@ class PermissionSeeder extends Seeder
      */
     public function run()
     {
-        //
+        $permissions=[
```

Comment:
add underscore in permission name also add group in permissions table

Reply:
Ok